Vagrantfile

```diff
@@ -52,16 +52,20 @@ def vm_cpus
 end
 
 Vagrant.configure("2") do |config|
-  # always use Vagrants insecure key
+  # always use Vagrant's insecure key
   config.ssh.insert_key = false
-
   config.vm.box = "coreos-%s" % $update_channel
-  config.vm.box_version = ">= 681.2.0"
-  config.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json" % $update_channel
+  config.vm.box_version = "= 647.2.0"
```

Comment:
Hmm. Since the beta/alpha jobs just want to use the latest version on their respective channels, maybe we can make this a switch flag if the channel == stable. Would that be a reasonable workaround?